packages/dappmanager/src/modules/ethClient/changeEthMultiClient.ts

```diff
@@ -43,3 +61,80 @@ export async function changeEthMultiClient(
     eventBus.runEthClientInstaller.emit();
   }
 }
+
+// Utils
+
+export async function migrateEthClientFullNode(
```

Comment:
`migrate` is a super generic word that doesn't mean anything. Try to find a better name to summarize the purpose of this function

https://medium.com/swlh/how-to-better-name-your-functions-and-variables-e962a4ef335b